### PR TITLE
User admin panel: link group memberships to groups

### PR DESCRIPTION
**Pull request by mhjb**
image

Very minor — needs smoke test @gdpelican ?

### CODE REVIEW

**Comment by robguthrie** [comment]
I don't think this needs a smoke test.

File: app/admin/users.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def find_resource
       table_for user.memberships.each do |m|
         column :group_id
         column :group_name do |g|
-          Group.find(g.group_id).full_name
+          link_to Group.find(g.group_id).full_name, [:admin, Group.find(g.group_id)]
```
Comment (gdpelican) [inline app/admin/users.rb]: I really don't want to make 2 db calls inside a table here.
```
  group = g.group # I think g is a membership here?
  link_to group.full_name, [:admin, group]
```

**Comment by mhjb** [comment]
Thanks for that @gdpelican — I couldn't quite figure it out. Amending commit now.

**Comment by gdpelican** [comment]
Sweet, thanks! :green_apple:

**Pull request merged**